src/render/smcat/index.js

```diff
@@ -1,8 +1,8 @@
-const Handlebars = require("handlebars/dist/handlebars.runtime");
-const _clonedeep = require("lodash.clonedeep");
+import Handlebars from "handlebars/dist/handlebars.runtime.js";
+import _clonedeep from "lodash.clonedeep";
 
-/* eslint import/no-unassigned-import: 0 */
-require("./smcat.template");
+// eslint-disable-next-line no-unused-vars
```

Comment:
Explain why we’re doing this. And research a bit what is going on exactly (see above)